addons/hr_recruitment/tests/test_utm.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo.exceptions import AccessError
+from odoo.tests.common import new_test_user
+
```

Comment:
Now we will have repeated imports,

Instead -lines 4-6

![image](https://github.com/odoo/odoo/assets/11148685/ce10850d-1f30-491b-8398-c2549664f307)

Reply:
Tks, I've revised.